src/controllers/QueryController.ts

```diff
@@ -665,7 +686,7 @@ export class QueryController extends RootComponent {
     let graph = Dom.createElement('div', { className: 'coveo-debug-durations' });
     let debugRef = BaseComponent.getComponentRef('Debug');
     dom.appendChild(graph);
-    _.forEach(debugRef.durationKeys, (key: string) => {
+    debugRef.durationKeys.forEach((key: string) => {
```

Comment:
One thing to watch out for with these changes is whether the property (`durationKeys`) can be `null` or `undefined`. If it's possible, the underscore `forEach` method will handle these cases, whereas `<property>.forEach` would throw an error when the property is falsey.

Reply:
Hmm noted